dashdotdb/models/dashdotdb.py

```diff
@@ -13,6 +13,25 @@ class Token(db.Model):
     serviceslo = db.relationship('ServiceSLO', backref='token')
 
 
+class Tokens(db.Model):
+
+    __tablename__ = 'tokens'
+
+    id = db.Column(db.Integer, primary_key=True)
+    uuid = db.Column(db.String(36))
+    data_type = db.Column(db.String(20))
```

Comment:
this should be an enum, only allowing certain values, not a random string

Reply:
Done